src/stores/exitHandler.ts

```diff
@@ -110,10 +132,20 @@ export default class ExitHandler extends ContractStore {
     return tx;
   }
 
+  watchExits = () => {
+    this.iContract.events.ExitStarted({fromBlock: 0})
+    .on('data', async (event) => {
+      const utxoId = new Outpoint(event.returnValues.txHash, parseInt(event.returnValues.outIndex)).getUtxoId();
+      const exit = await this.iContract.methods.exits(this.web3.root.instance.utils.toHex(utxoId)).call();
+      this.Exits.push({ event, exit, suspect: false });
+    })
+  }
+
   @autobind
   @action
   private setAddress() {
```

Comment:
You don't need this method. To have `address` property you can defined computed getter

```es6
@computed
get address() {
  return this.plasmaConfig.exitHandlerAddr;
}
```

and change reaction in constructor to
```
reaction(this.address, this.watchExits)
```